examples/mem-alloc-test/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
```

Comment:
This crate isn't tested in CI. Can you add it?

Reply:
How do I add to CI?